### PR TITLE
feat: add customer portal to user resolver

### PR DESCRIPTION
**Pull request by Carolusian**

### CODE REVIEW

File: src/connectors/stripe/index.ts [hunk 1]
```diff
@@ -246,6 +246,24 @@ class StripeService {
...
+   * Get customer portal URL
+   */
+  getCustomerPortal = async ({ customerId }: { customerId: string }) => {
+    try {
+      if (!environment.stripeReturnURL) {
+        throw new ServerError('matters stripe return URL has not been set')
+      }
+      const session = await this.stripe.billingPortal.sessions.create({
+        customer: customerId,
+        return_url: environment.stripeReturnURL,
```
Comment (robertu7) [inline src/connectors/stripe/index.ts:260]: I saw `return_url` is optional and can be configured in [Stripe Dashboard](https://dashboard.stripe.com/settings/billing/portal), would it more simple if use the latter instead?

**Review by robertu7 — CHANGES_REQUESTED** [review]
left a comment about the return URL, others LGTM.

File: src/connectors/stripe/index.ts (hunk 1 above)
Comment (Carolusian) [reply src/connectors/stripe/index.ts]: @robertu7 the `return_url` configuration is removed from the code as requested. pls check.

**Review by robertu7 — APPROVED** [review]
Let's try. ☃️

**Pull request merged**